src/main/java/io/stargate/sgv2/jsonapi/service/updater/DocumentUpdater.java

```diff
@@ -62,6 +78,7 @@ public DocumentUpdaterResponse apply(JsonNode readDocument, boolean docInserted)
    */
   private boolean update(ObjectNode docToUpdate, boolean docInserted) {
     boolean modified = false;
+    List<UpdateOperation> updateOperations = updateClause.buildOperations();
```

Comment:
As per earlier comment, should just pass `updateClause.buildOperations()` by caller, since it has been done earlier (it's not a cheap thing to do)